src/test/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortFilterRecordUnitTest.java

```diff
@@ -0,0 +1,72 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.GATKBaseTest;
+
+import org.broadinstitute.hellbender.utils.bigquery.AvroFileReader;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class ExtractCohortFilterRecordUnitTest extends GATKBaseTest {
+
+    // TODO refactor - this is copied from ExtractCohortRecordUnitTest
+    private static GenericRecord getSingleRecordFromAvroFile (String inputFileName) {
+        final AvroFileReader avroFileReader = new AvroFileReader(inputFileName);
+        return avroFileReader.next();
+    }
+
+    // load in test data
+    private static final String testFileDir = "src/test/resources/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortFilterRecord/";
+    private static final GenericRecord inputGenericRecord = getSingleRecordFromAvroFile(testFileDir + "test_input.avro");
+
+
+    private ExtractCohortFilterRecord allDefinedRecord = new ExtractCohortFilterRecord(inputGenericRecord);
+
+    @Test
+    public void testGetContig() {
+        String expectedContig = "chr1";
```

Comment:
is it preferred to define these single-use objects explicitly or just stick "chr1" into the assertion statement directly?

Reply:
either way is fine.  I tend to put them right in the assertion so it's easier to read if I'm not also using it to construct the test data.  In that case I pull it out so I don't have `"chr1"` repeated throughout the method

Reply:
yeah reading the rest of these tests I would just inline it into the assert for readability